hr_contract_operating_unit/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
```

Comment:
```suggestion
* ForgeFlow S.L. <contact@forgeflow.com>
```

Reply:
Hi @JuanyDForgeflow  Can you change Eficent by ForgeFlow everywhere in the module? Then I think it would be merged soon :)

Reply:
@AaronHForgeFlow Done